content/body/status.php

```diff
@@ -1,6 +1,6 @@
 <p>
   Status messages allow screen readers and other assistive technology to tell users that content has changed on the page 
-  without interrupting what the user is doing by changing element has focused.  A perfect example of this 
+  without interrupting what the user is doing by changing elements.  A perfect example of this 
```

Comment:
I think this sentence is more accurate if you specify that what is changing is the focus, not the element itself.
```suggestion
  without interrupting what the user is doing by changing which element has focus.  A perfect example of this 
```